appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -543,7 +545,9 @@ private YaVersion() {
   // - AboutScreen property was added
   // For FORM_COMPONENT_VERSION 13:
   // - The Screen.Scrollable property was set to False by default
-  public static final int FORM_COMPONENT_VERSION = 13;
+  // For FORM_COMPONENT_VERSION 14:
+  // - The Screen1.AppName was added.
+  public static final int FORM_COMPONENT_VERSION = 14;
```

Comment:
Make sure to add a comment saying the the AppName property was added and no block need to be change.  See the analogous comment in upgrading to versions 7, 8, and  9 above